api/src/main/java/jakarta/servlet/ServletOutputStream.java

```diff
@@ -272,10 +272,9 @@ public void println(double d) throws IOException {
     }
 
     /**
-     * This method can be used to determine if data can be written without blocking.
+     * Returns true if data can be written without blocking else returns false.
```

Comment:
Probably we should also say something about the scheduling ramifications of a false return

Reply:
+1 - for both `ServletInputStream` and `ServletOutputStream`. Do you have some text in mind? A new PR is probably the simplest way to proceed.

Reply:
will try to do a PR soon.